### ISSUE TITLE
Code Syntax on File activation not checked, when Roslynator is activated.

### ISSUE THREAD

**Issue by mz-zmsoft** [JosefPihrt/Roslynator#627]
**Product and Version Used**: 
VSCODE 1.41.1 on OSX
josefpihrt-vscode.roslynator 2.3.0
ms-vscode.csharp 1.21.9

**Steps to Reproduce**:
- Open any C# based project
- Create a new folder / namespace in this project called "Demo"
- Create a new Class within Demo Folder "Class1".
- Open Class1.cs  and add following line:
```csharp
  private Class2 demo = null;
```
- Use Intellisense for Class2 File Creation ("Generate class Class2 in separate File")
- Open the new created File "Class2.cs" and save it!
- Change back to File "Class1.cs".

![image](https://user-images.githubusercontent.com/55440373/71640216-ec014480-2c85-11ea-84b1-58616fceade3.png)

- ```Class2``` keyword is still red marked. Even when saving "Class1.cs" the red error remains.
- Only way to resolve this issue is to edit the file on any position (insert space or whatever). It seems like a "file input" event will force a re-check of the syntax. The file check succeeds now.

![image](https://user-images.githubusercontent.com/55440373/71640235-83669780-2c86-11ea-98b8-388ee6fbce1d.png)

**Actual Behavior**:
Syntax check isn't triggered, when a source file is focused.

**Expected Behavior**:
This is the behavior, when Roslynator Extension is deactivated (tested the same way like above)...
Syntax check is triggered immediately, when a source file is focused.

**Comment by JosefPihrt** [comment]
Hi,

Roslynator does not manage how the code analysis is executed. OmniSharp is responsible for that.

I think that relevant repos are https://github.com/OmniSharp/omnisharp-roslyn or https://github.com/OmniSharp/omnisharp-vscode.

By the way, refactoring "Generate class X in separate file" is not Roslynator refactoring, It is from Microsoft.

**Comment by mz-zmsoft** [comment]
Thanks for feedback,

I'll create a issue at omnisharp-roslyn repository.

I know, "Generate class X in separate file" is not a a Roslynator refactoring. That should only help to reproduce my problem.
